packages/models-library/src/models_library/api_schemas_webserver/resource_usage.py

```diff
@@ -131,7 +131,7 @@ class UpdatePricingUnitBodyParams(InputSchema):
     unit_extra_info: UnitExtraInfo
     default: bool
     specific_info: SpecificInfo
-    pricing_unit_cost_update: PricingUnitCostUpdate | None
+    pricing_unit_cost_update: PricingUnitCostUpdate | None = Field(default=None)
```

Comment:
and here as well

Reply:
Here its fine to leave default value None. (also FYI this endpoint is used just internally by us "admin" users)